ui/src/components/DownloadButton/OpenApiDownloadBtn.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { createRESTURL } from '@splunk/splunk-utils/url';
+import { app } from '@splunk/splunk-utils/config';
+import Icon from '@splunk/react-icons/ArrowBroadUnderbarDown';
+import DownloadButton from './DownloadButton';
+
+function OpenApiDownloadButton() {
+    return (
+        <div style={{ float: 'right' }}>
```

Comment:
1. Please decouple layout and component responsibilites The button component should not be responsible for layout. 
2. Avoid floats in 2023 for layout: https://css-tricks.com/is-css-float-deprecated/

Reply:
i used 
```
        display: flex;
        justify-content: space-between;
```
in parent element instead of that float value.